addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkUtils.java

```diff
@@ -167,4 +167,25 @@ public static boolean nativePing(String hostname, int port, int timeout)
         return exitValue == 0;
     }
 
+    public static boolean arping(String hostname, int timeout, String networkInterface)
+            throws InvalidConfigurationException, IOException, InterruptedException {
+        Process proc;
+        if (SystemUtils.IS_OS_UNIX) {
+            proc = new ProcessBuilder("arping", "-I", networkInterface, "-w", String.valueOf(timeout / 1000), "-c", "1",
+                    hostname).start();
+        } else {
+            throw new InvalidConfigurationException("Arping not supported");
+        }
+
+        int exitValue = proc.waitFor();
+        if (exitValue != 0) {
+            throw new IOException("Ping stopped with Error Number: " + exitValue + " on Command :" + "ping"
+                    + (SystemUtils.IS_OS_UNIX ? " -I " + String.valueOf(networkInterface) : "")
+                    + (SystemUtils.IS_OS_UNIX ? " -t " : " -w ")
+                    + (SystemUtils.IS_OS_UNIX ? String.valueOf(timeout / 1000) : String.valueOf(timeout))
+                    + (SystemUtils.IS_OS_UNIX ? " -c" : " -n") + " 1 " + hostname);
+        }
```

Comment:
Not at this moment. I copied it from the nativePing function and left it there so it might be easier to add this functionality for Windows. Should I remove it?